ConvivaTestApp/src/androidTest/java/com/bitmovin/analytics/conviva/testapp/PlaybackControlsTrackingTests.kt

```diff
@@ -25,16 +25,14 @@ class PlaybackContorlsTrackingTests: TestBase() {
 
         // load source and verify
         loadSource(activityScenario, DEFAULT_DASH_VOD_SOURCE)
-
         verifyPlaying(activityScenario = activityScenario)
 
         // pause playback
         activityScenario.onActivity { activity: MainActivity ->
             clearMocks(playerStateManagerMock!!)
             activity.bitmovinPlayer.pause()
-            Thread.sleep(2000)
         }
-
+        Thread.sleep(2000)
```

Comment:
Why do we need the `Thread.sleep(2000)`'s in here? Is there no better way? Is "it" (whatever we're waiting for) guaranteed to be available/done within 2000ms?

Reply:
The wait is for the action like load(), play(), pause() etc. to be finished and expected event to be triggered by player, captured by Conviva integration code and expected Conviva API call executed. 

The value of 2 seconds was arrived at by multiple test runs. Some operations finish in lesser time but I used 2 seconds sleep in most cases as this resulted in stable test results and also does not add lot of delay to test case execution time.

Reply:
Can't we wait for one (or more) player events or so instead? But I just realized this very similar discussion above with Lorenz. We can keep the discussion there, I'll resolve this one here.